### ISSUE TITLE
low priority - fix spelling errors?

### ISSUE THREAD

**Issue by arcanor** [alvyxaz/barebones-masterserver#94]
I realize this is a low priority, but it bugs the QA Tester inside me when things are misspelled. :)

In particular, here are some examples that I'd love to see fixed in the base framework code:

- _IIncommingMessage_ should become _IIncomingMessage_
- in **MsfAuthClient**: __isLogginIn_ should become __isLoggingIn_

**Comment by alvyxaz** [comment]
Oh man! I should have known that it's `IIncomingMessage`.

It would be a huge change, as it's used everywhere, and it will break a lot of custom code of the developers.
 
Either way, the "break" is small, and I think it should be done either way, just not sure when's the best time to do it, haha. I'll try to squeze it in after the V2.0.4 ;)

**Comment by arcanor** [comment]
Yeah, it seems like a lot of change for a small benefit. So, "whenever". :)
